### PR TITLE
Portfolio: Added details popup window

### PR DESCRIPTION
**Pull request by amare1990**
- Modify the project sections part of the markup page to do using javascript
- Create javascript to store resources of the project
- Modify javascript for the mobile version popup window
- Modify CSS styles for the original page of the mobile and desktop version page
- Modify CSS styles for the popup window for the mobile and desktop version
- Modify javascript to stick to Figma design
- Modifyu markup, CSS styles and javascript to stick to Figma design

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -579,6 +275,11 @@ <h2 class="name-welcome">
     </ul>      
   </div>
   <!--     Mobile Menu  Ends here      -->  
+  <!-----------Popup window here starts-->
+  <div class="popup-card-wrapper hide">
+
+  </div>
+  <!-----------Popup window ends here-->
  <script src="/js/scripts.js"></script>
```
Comment (ghost) [inline index.html]: - [ ] Kindly work on the responsiveness and design of your pop-up windows to match the Figma design. They should fit the mobile and desktop versions properly.
 ### Your design

image

image


###Figma design

image

image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

YesYouAreGIF (2)

## Highlights

✅ No linter error
✅ README is okay
✅Pop up window works as expected👍


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [OPTIONAL] Your `see project` buttons on the project section appear to be wrongly positioned. Give them a `margin-top`  to have a good spacing of elements

### Your design

image

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
```
Comment (Kamba56) [inline .vscode/settings.json:2]: - [Optional] Please set up linters for javascript by copying the files and folders from [this](https://github.com/microverseinc/linters-config/tree/master/javascript) repo to your own repo and run `npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x` to install eslint.

File: js/scripts.js [hunk 3]
```diff
@@ -79,4 +78,385 @@ document.querySelector('.contact_mobile').addEventListener('click',() =>{
...
+      e.addEventListener('click', function() {
+        const index = e.getAttribute('whichProjectBtn');
+        popupWindow.classList.toggle('hide');
+        const experiencePopup = projects[index].experiences;
+        const techpopup = projects[index].technologies;
+        let li1 = '';
+        for (let i = 0; i < experiencePopup.length; i++) {
+          li1 += `
+            <li class="popup-exp-name">${experiencePopup[i]}</li>
+          `;
```
Comment (Kamba56) [inline js/scripts.js:199]: - [ ] Please kindly fix the pop-up window. I noticed it is not coming up at all when the `see project` button is clicked.

File: js/scripts.js [hunk 4]
```diff
@@ -79,4 +78,385 @@ document.querySelector('.contact_mobile').addEventListener('click',() =>{
...
+            </div>
+          </div>
+        </div> `;
+
+        
+        
+          /* const popupCardWrapper = document.querySelector('.popup-card-wrapper');
+             const popupCard = document.querySelector('.popup-card');
+             popupWindow.append(popupCard);
+          */
```
Comment (Kamba56) [inline js/scripts.js:239]: - [Optional] kindly remove all unused code from your documents. It is best practice not to leave commented/unused code in your documents.

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @amare1990, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- [x] Great use of GitHub flow
- [x] Great commit messages

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/scripts.js (hunk 3 above)
Comment (amare1990) [reply js/scripts.js:199]: Hi @Kamba56.
How are you? I used Firefox browser, VS code live server. I tested it for both the mobile version and desktop version. When I click the 'See project' button for each project and for both desktop version, it works. Kindly see it again please.

**Review by DammyShittu — APPROVED** [review]
Hi @amare1990,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**